fenix/app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -386,6 +394,11 @@ class SettingsFragment : PreferenceFragmentCompat() {
             resources.getString(R.string.pref_key_nimbus_experiments) -> {
                 SettingsFragmentDirections.actionSettingsFragmentToNimbusExperimentsFragment()
             }
+            resources.getString(R.string.pref_key_install_local_addon) -> {
+                // TODO: Report telemetry
```

Comment:
why?

Reply:
Should we collect info about how offence users use the option?

Reply:
maybe yeah but that'd be a separate bug/ticket (that we need to file)

Reply:
I'll file a separated ticket for it!